index.html

```diff
@@ -68,7 +68,340 @@ <h3>
     </div>
     </div>
     
-    </section>
+</section>
+
+<section class="recent-works" id="portfolio-section">
+   
+
+  
+  <header class="title-header">
+
+     
+      <h2 class="title">My Recent Works</h2>
+  
+<div><hr class="title-border"></div>
+    </header>
+     <div class="d-flex">
+    <div class="work">
+        <section >
+          <img src="images/bestwork.png" class="desk-img" alt="Best work">
```

Comment:
- [ ] kindly add the correct margin from the right side of the card like shown in figma
- [ ] the button and the technologies list should be aligned together to the left , kindly fix that

![image](https://user-images.githubusercontent.com/89282221/192110785-eb717613-7bfb-4ee2-a129-f862c9d1ec6b.png)